src/main.ts

```diff
@@ -165,6 +169,45 @@ const command: Command = {
 		else {
 			return compile(config(configArgs), options);
 		}
+	},
+	eject(helper: Helper) {
+		const ejectOutput: EjectOutput = {
+			npm: {
+				devDependencies: {
```

Comment:
Well either way we are going to have a list to maintain. Granted, a list of excluded dependencies will be shorter (for now) than a list of included dependencies, but I'm not sure if that's just a step sideways. I like the idea of having just the one list, I'm just not quite sure how to pull it off because the `dependencies` section has a handful of packages that aren't in this list (`cldr-data`, `@dojo/shim`/`core`/`i18n`, `has`).